pycbc/workflow/plotting.py

```diff
@@ -417,9 +419,11 @@ def make_results_web_page(workflow, results_dir, explicit_dependencies=None):
     node.add_opt('--template-file', template_path)
     workflow += node
     if explicit_dependencies is not None:
+        import Pegasus.DAX3 as dax
```

Comment:
Here as well, we're not using pegasus4 any more.

Reply:
Thanks, @spxiwh.  I looked at the changes to this file, and some of them, e.g., the one above, predate PR #3826, so they should not be there.  Perhaps there was an oversight in fixing conflicts when I rebased.  In any event, I undid these changes.

Reply:
By the way, is 'depend**a**ncy' intentional in the various 'add_*_dependancy'?

Reply:
Whoops, that was a spelling mistake by me, but it is what the function is currently called https://pycbc.org/pycbc/latest/html/pycbc.workflow.html#pycbc.workflow.pegasus_workflow.Workflow.add_explicit_dependancy